chef_master/source/config_rb_server_optional_settings.rst

```diff
@@ -1022,6 +1022,18 @@ This configuration file has the following settings for ``opscode-erchef``:
 ``opscode_erchef['port']``
    The port on which the service is to listen. Default value: ``8000``.
 
+``opscode_erchef['reindex_batch_size']``
+  The number of items to fetch from the database and send to the search index at a time. Default Value: ``10``.
+
+``opscode_erchef['reindex_sleep_min_ms']``
+  The minimum number of milliseconds to sleep before retrying a failed attempt to index an item. Retries are delayed a random number of miliseconds between ``reindex_sleep_min_ms`` and ``reindex_sleep_max_ms``. Set both this and ``reindex_sleep_max_ms`` to 0 to skip the sleeping between retries. Default Value: ``500``
+
+``opscode_erchef['reindex_sleep_max_ms']``
+  The maximum number of milliseconds to sleep before retrying a failed attempt to index an item. Retries are delayed a random number of miliseconds between ``reindex_sleep_min_ms`` and ``reindex_sleep_max_ms``. Set both this and ``reindex_sleep_min_ms`` to 0 to skip the sleep between retries. Default Value: ``2000``
```

Comment:
Above, it's "skip the sleeping". I doubt this matters, and it might even have been intentional to keep the prose non-repetitive :wink: (if so -- :+1:)